src/_utils/TradeUtils.js

```diff
@@ -20,3 +20,50 @@ export const digitOptions = (from, to) => {
     const arr = Array.from(tArr);
     return arr.map((i, idx) => idx + from).map(x => ({ value: x, text: x.toString() }));
 };
+
+export const durationText = unit => {
+    switch (unit) {
+        case 't': return 'Ticks';
+        case 's': return 'Seconds';
+        case 'm': return 'Minutes';
+        case 'h': return 'Hours';
+        case 'd': return 'Days';
+        default: return null;
```

Comment:
There are no clear guidelines really about both.
The two shouldn't have existed together really.

In JSX null is used often, but in pure code, I find it is very rarely needed, and since undefined exists anyways, I try not to use it.